.env.example

```
@@ -39,6 +39,7 @@ TOGETHER_API_KEY=
 MINIMAX_API_KEY=
 MODULATE_API_KEY=
 LMNT_API_KEY=
+FLUXIONS_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FLUXIONS_API_KEY key should go before the GLADIA_API_KEY key

(UnorderedKey)